Cargo.toml

```diff
@@ -39,11 +42,11 @@ codegen-units = 1
 lto = true
 
 [workspace.dependencies.soroban-sdk]
-version = "0.4.1"
+version = "0.4.3"
```

Comment:
@dmkozh , should the sdk version be v0.5.0 now?

Reply:
Sure, this PR was created before the SDK version has been bumped.